tests/beaker_tests/cisco_interface/routedintf_provider_negatives.rb

```diff
@@ -510,39 +526,41 @@
     logger.info("Check interface instance absence on agent :: #{result}")
   end
 
-  # @step [Step] Requests manifest from the master server to the agent.
-  step 'TestStep :: Get negative test resource manifest from master' do
-    # Expected exit_code is 0 since this is a bash shell cmd.
-    on(master, RoutedIntfLib.create_channel_group_manifest_negative)
-
-    # Expected exit_code is 1 since this is a puppet agent cmd with error.
-    cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
-      'agent -t', options)
-    on(agent, cmd_str, acceptable_exit_codes: [1])
-
-    logger.info("Get negative test resource manifest from master :: #{result}")
-  end
-
-  # @step [Step] Checks cisco_interface resource on agent using resource cmd.
-  step 'TestStep :: Check cisco_interface resource absence on agent' do
-    # Expected exit_code is 0 since this is a puppet resource cmd.
-    # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
-      "resource cisco_interface 'ethernet1/4'", options)
-    on(agent, cmd_str) do
-      search_pattern_in_output(stdout,
-                               { 'channel_group' => RoutedIntfLib::CHANNEL_GROUP_NEGATIVE },
-                               true, self, logger)
-    end
-
-    logger.info("Check cisco_interface resource absence on agent :: #{result}")
-  end
+  # TBD: CHANNEL-GROUP NEGATIVE TEST DOES NOT RETURN EXIT CODE [1]
```

Comment:
Unrelated bug with channel-group. This is a minor negative test so I'm removing it for now, will create a bug entry for the backlog.